types/OrderDto.ts

```diff
@@ -4,27 +4,44 @@ import { UserDto } from "./UserDto"
 
 export type OrderDto = {
```

Comment:
actually since we are changing this right would it be good to just make OrderDto contain all the columns of the table, then if we need like just specific columns we can do Partial<OrderDto>, so that this is reusable everywhere